core/domain/user_jobs_one_off.py

```diff
@@ -18,13 +18,9 @@
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
 import ast
-import logging
```

Comment:
I've removed this because this job was temporary and was failing when I tried upgrading the version from 34 to 35 in the tests for this job. This is in order to mirror the changes made by @donosco98 as seen in [this commit](https://github.com/oppia/oppia/pull/9985/commits/89342471240d7598c2c1654b68fc7d37df14fca0).